src/stores/EndpointConfig.tsx

```diff
@@ -215,6 +215,14 @@ const getInitialState = (
         set(
             produce((state) => {
                 state.endpointSchema = val;
+
+                const { endpointConfig, endpointCustomErrors } = get();
+
+                populateEndpointConfigErrors(
+                    endpointConfig,
+                    endpointCustomErrors,
+                    state
+                );
```

Comment:
This should be good and is a funny side effect of how things run.

When the endpoint schema is being set there should be no `endpointConfig` possible so we could honestly just pass an empty default here. However, I think we're fine just using whatever was last in the store. When switching connectors passing an old config here isn't an issue because right after this call we'll set the `endpointConfig` and then call the `populateEndpointConfigErrors` function.